moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -2039,28 +2039,40 @@ double RobotState::testRelativeJointSpaceJump(const JointModelGroup* group, std:
 double RobotState::testAbsoluteJointSpaceJump(const JointModelGroup* group, std::vector<RobotStatePtr>& traj,
                                               double revolute_threshold, double prismatic_threshold)
 {
-  bool test_revolute = revolute_threshold > 0;
-  bool test_prismatic = revolute_threshold > 0;
+  struct
+  {
+    double limit;
+    bool check;
+  } data[2] = { { revolute_threshold, revolute_threshold > 0.0 }, { prismatic_threshold, prismatic_threshold > 0.0 } };
```

Comment:
how is this more readable?? this should be reverted.

Reply:
I don't see the value in these changes to `robot_state.cpp`. They are inconsistent with the rest of MoveIt! and are unnecessarily complex.

Reply:
I do think, that the source below is more readible than before.
Initialization of the array struct could be improved by disabling clang...